### PR TITLE
Create timberland.md

### LINKED ISSUE #209: Add data for Timberland
#### Description

Add a file with data for Timberland
https://www.instagram.com/p/CA3kcWTjUo5

Use `companies-on-blm/examples/add-a-company.md` as a reference for how to add data for a company. If information for a field cannot be found, please delete that line.

#### Helpful tips

Image: Save the image from a tweet, instagram, or Facebook post and rename it to the company name and place in the `images` folder. Reference the name of the file in the `image` field for the company you're working on.

Fortune Rank: Search for the company [here](https://fortune.com/fortune500/2020/search/). If they aren't ranked by Fortune, take this field out.

Twitter: Use the [advanced search](https://www.twitter.com/search-advanced) feature.
First, find the Twitter handle of the company (sometimes there are several) & CEO. Googling `twitter account <company name>` usually works.
Replace `zillow` in the following url with the name of the handle you are searching for.

https://twitter.com/search?q=(from%3Azillow)%20min_faves%3A10%20until%3A2020-06-05%20since%3A2020-05-28%20-filter%3Areplies

How this url works:

- `(from%3Azillow)%20`: from the @zillow account
- `min_faves%3A10%20`: minimum 10 likes
- `since%3A2020-05-28%20`: since May 28, 2020
- `until%3A2020-06-05%20`: until June 5, 2020
- `-filter%3Areplies`: filter replies

You can change these values in the URL to expand or narrow your search.

Google:

The following Google searches are helpful for finding statements & relevant articles about a company:

- `<company name> Black Lives Matter`
- `<company name> George Floyd`
- `<company name> Diversity`
- `<company name> Inclusion`

Instagram:

Google `<company name> Instagram` to find their account and scroll down towards the end of May/early June. The statement is generally a black background.

LinkedIn:

Google `<company name> LinkedIn` to find their account and locate a statement towards the end of May/early June.

### PR DESCRIPTION
**Pull request by nycbeardo**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #209 

## Type of change

Please delete options that are not relevant.

- [x] Company data addition

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Browser: Google Chrome

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: companies/timberland.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    date_posted: 2020-05-31
+    type: Instagram
+    raw_text: >
+        In 1992 we shared a message. Sadly, we bring it back today. We don’t want to. We shouldn’t have to. As humans, we have the responsibility to care for our planet, ourselves, and each other—all of us, no matter who you are. Let’s work together now for a better future. Caring is no longer enough.Use your voice. 
+        Take action at colorofchange.org
+- url: https://twitter.com/Timberland/status/1267220953131606017
+    date_posted: 2020-05-31
+    type: Tweet
+    raw_text: >
+        In 1992 we shared a message. Sadly, we bring it back today. We have the responsibility to care for our planet, ourselves + each other no matter who you are. Let’s work together for a better future. Caring is no longer enough.
```
Comment (kelseyhuse30) [inline companies/timberland.md:15]:
```suggestion
        In 1992 we shared a message. Sadly, we bring it back today. We have the responsibility to care for our planet, ourselves + each other no matter who you are. Let’s work together for a better future. Caring is no longer enough.
```

File: companies/timberland.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+name: Timberland
+tags: ["outdoors"]
+blm_statements:
+- url: https://www.instagram.com/p/CA3kcWTjUo5/
+    date_posted: 2020-05-31
+    type: Instagram
+    raw_text: >
+        In 1992 we shared a message. Sadly, we bring it back today. We don’t want to. We shouldn’t have to. As humans, we have the responsibility to care for our planet, ourselves, and each other—all of us, no matter who you are. Let’s work together now for a better future. Caring is no longer enough.Use your voice.
```
Comment (kelseyhuse30) [inline companies/timberland.md:9]:
```suggestion
        In 1992 we shared a message. Sadly, we bring it back today. We don’t want to. We shouldn’t have to. As humans, we have the responsibility to care for our planet, ourselves, and each other—all of us, no matter who you are. Let’s work together now for a better future. Caring is no longer enough.Use your voice.
```

File: companies/timberland.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+name: Timberland
+tags: ["outdoors"]
+blm_statements:
+- url: https://www.instagram.com/p/CA3kcWTjUo5/
```
Comment (kelseyhuse30) [inline companies/timberland.md:5]:
```suggestion
  - url: https://www.instagram.com/p/CA3kcWTjUo5/
```

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "cSpell.ignoreWords": [
```
Comment (kelseyhuse30) [inline .vscode/settings.json:2]: 👀 what's cSpell?

File: companies/timberland.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    date_posted: 2020-05-31
+    type: Instagram
+    raw_text: >
+        In 1992 we shared a message. Sadly, we bring it back today. We don’t want to. We shouldn’t have to. As humans, we have the responsibility to care for our planet, ourselves, and each other—all of us, no matter who you are. Let’s work together now for a better future. Caring is no longer enough.Use your voice. 
+        Take action at colorofchange.org
+- url: https://twitter.com/Timberland/status/1267220953131606017
+    date_posted: 2020-05-31
```
Comment (kelseyhuse30) [inline companies/timberland.md:12]:
```suggestion
  date_posted: 2020-05-31
```

make sure url, date_posted, type, raw_text are all vertically aligned

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]
Looking good, thanks for finding all the info including that additional Tweet! Just a few formatting issues to clean up, let me know if they don't seem resolvable.

Also, please remove the `.vscode/settings.json` file since those are your individual vscode settings.  You can add `.vscode` to the `.gitignore` file instead if you want, to prevent anyone from committing that folder to our repo.

**Comment by nycbeardo** [comment]
Wanted to know if the changes are acceptable

**Review by kelseyhuse30 — APPROVED** [review]
Thank you @nycbeardo ! 👏🏻 💓

**Pull request merged**